### DISCUSSION TITLE
Unit 7, Quiz 7.2 part 5

### DISCUSSION THREAD

**Discussion by gracecluvohio** [Lightning-AI/dl-fundamentals#90] [Lecture videos and quizzes] [answered]
![image](https://github.com/Lightning-AI/dl-fundamentals/assets/94275517/e9135fda-8042-444f-9ab3-1ce2a1c13019)
Is the answer not 113? The padding should be multiplied by 2 according to the slides so [(224-3+2*2)/2] + 1 = 113

**Comment by rasbt** [comment, marked as answer]
You are absolutely right. Thanks for raising that. Just updated it.

We can also double check it via

```python
from torch.nn import Conv2d
import torch

conv2d = Conv2d(1, 1, kernel_size=3, stride=2, padding=2)

img = torch.rand(1, 224, 224)
conv2d(img).size()
```
